src/data/multiset/pi.lean

```diff
@@ -50,6 +50,25 @@ begin
   all_goals { simp [*, pi.cons_same, pi.cons_ne] },
 end
 
+@[simp]
+lemma pi.cons_eta {m : multiset α} {a : α} (f : Π a' ∈ a ::ₘ m, δ a') :
+  pi.cons m a (f _ (mem_cons_self _ _)) (λ a' ha', f a' (mem_cons_of_mem ha')) = f :=
+begin
+  ext a' h',
+  by_cases a' = a,
+  { subst h, rw [pi.cons_same] },
+  { rw [pi.cons_ne _ h] }
+end
+
+lemma pi.cons_injective {a : α} {b : δ a} {s : multiset α} (hs : a ∉ s) :
+  function.injective (pi.cons s a b) :=
```

Comment:
```suggestion
  injective (pi.cons s a b) :=
```
`function` is already open, I believe.

Reply:
Given I just moved these lines and I've already done the forward-port, I'm inclined to leave it as is

Reply:
Fine, I guess that doesn't matter much.

maintainer merge